lib/galaxy/authnz/psa_authnz.py

```diff
@@ -23,14 +23,16 @@
     'google': 'social_core.backends.google_openidconnect.GoogleOpenIdConnect',
     'globus': 'social_core.backends.globus.GlobusOpenIdConnect',
     'elixir': 'social_core.backends.elixir.ElixirOpenIdConnect',
-    'okta': 'social_core.backends.okta_openidconnect.OktaOpenIdConnect'
+    'okta': 'social_core.backends.okta_openidconnect.OktaOpenIdConnect',
+    "fence": "social_core.backends.fence.Fence"
```

Comment:
```suggestion
    'fence': 'social_core.backends.fence.Fence'
```

Reply:
(super minor, just standardizing the quotes like everything else in this list)

Reply:
Thanks.